crates/uv-cli/src/lib.rs

```diff
@@ -3562,7 +3562,7 @@ pub struct IndexArgs {
     /// indexes.
     ///
     /// If a path, the target must be a directory that contains packages as wheel files (`.whl`) or
-    /// source distributions (`.tar.gz` or `.zip`) at the top level.
+    /// source distributions (usually `.tar.gz` or `.zip`) at the top level.
```

Comment:
Initially wanted to also add a link to the new section here, but the code generating markdown pages output the links with `<a>` HTML tags, which made it not possible to have both a correct link and `mkdocs` not complaining about the link not existing in strict mode. Since there are no other links to other pages here and in `settings.rs`, I assumed that this is simply not yet supported by the code generating markdown pages, so left this out for now.